collections/concepts/access-controls.md

```diff
@@ -12,16 +12,266 @@ width: 1200
+Some business units may not want other parts of the organization to have direct access to data they own and manage. 
+Creating a Tenant for each business unit enables your organization to isolate and limit access to their data by
+issuing an Application with limited access controls to other units within the organization.
+
+<img src="/assets/images/concepts/tenant_separate_data.png">
+
+
+## What are Applications?
+
+An Application represents an external software system or service that will be integrated with the Basis Theory API.
+Basis Theory’s Applications are not coupled to a specific implementation type or technology.
+Applications could be a native app on a mobile device, a single-page application in a browser, or a backend service
+that executes on a server or in a serverless environment.
+
+Applications within Basis Theory have the following properties and characteristics:
+
+### Name
+
+A short description of the Application describing its intended use, for example "Acme Billing App" or "Acme Mobile App".
+
+### Application Type
+
+- **Private**: Used by a backend system or service that runs primarily on a server or in a serverless environment (e.g. NodeJs, .NET, Java, or Ruby). Private Applications allow the broadest access to Token data as the API Key is expected to be kept private and secured within your systems.
+
+- **Public**: Used for Browser, Mobile, or Desktop applications running natively on a device (e.g. iOS, Android, Windows, etc) where the application is collecting data for tokenization via an API call or using Basis Theory Elements. Public Applications have limited permissions available to ensure the API Key can be safely used within publicly available code.
+
+- **Management**: Used with scripts or the Basis Theory Terraform ([request access here](https://basistheory.com/contact)) modules to manage your [Tenant’s](https://developers.basistheory.com/concepts/tenants/) settings and services (Applications, Reactors, Proxy, etc) without logging into our Portal.
+
+### Access Controls
+
+An Application is granted access to your Tenant's resources in one of two ways: by granting the Application a set of 
+Permissions, or for more advanced scenarios, by defining one or more Access Rules.
+
+#### Permissions
+
+For basic authorization scenarios, an Application can simply be granted one or more permissions. 
+Each Application Type allows a different set of permissions, and you can find a detailed list of all of our 
+permissions for all of our Application Types [here](https://docs.basistheory.com/api-reference/#permissions-permission-types).
+
+Token permissions granted to Private or Public applications are applied to all Tokens 
+(i.e. they are not scoped to a particular Container of Tokens). In order to ensure sensitive data is not inadvertently 
+revealed by an Application, an implicit [transform](/concepts/access-controls#what-are-access-rules-transform)
+is applied for each operation:
+
+| Permission   | Transform |
```

Comment:
is there a way to center the row values? it doesn't look good right now

Reply:
I left aligned the header, but the table is way too big. This is just how they're styled in our Jekyll theme..